### PR TITLE
JS: fix azure performance

### PR DESCRIPTION
**Pull request by esben-semmle**
I enountered reproducible timeouts for `azure-sdk-for-node` on master, this change seems to fix it ([small evaluation](https://git.semmle.com/esben/dist-compare-reports/tree/js/semver-lib_1556263742267)). I am currently doing a full run on a clean master ([sneak peek](https://git.semmle.com/esben/dist-compare-reports/tree/js/fix-azure-performance_1556309687861)).

### CODE REVIEW

File: javascript/ql/src/semmle/javascript/frameworks/SocketIO.qll [hunk 1]
```diff
@@ -411,6 +411,11 @@ module SocketIOClient {
     exists(DataFlow::TypeTracker t2 | result = socket(invk, t2).track(t2, t))
   }
 
+  /**
+   * Gets the NPM package that countains `nd`.
```
Comment (xiemaisi) [inline javascript/ql/src/semmle/javascript/frameworks/SocketIO.qll]:
```suggestion
   * Gets the NPM package that contains `nd`.
```

**Comment by xiemaisi** [comment]
How is the evaluation looking?

**Comment by esben-semmle** [comment]
The full evaluation is 96% done.
`azure-sdk-for-node` is the only project with noteworthy timing changes (37 timeouts vs 0 timeouts).

**Comment by esben-semmle** [comment]
[Evaluation over](https://git.semmle.com/esben/dist-compare-reports/tree/js/fix-azure-performance_1556526393383): no other changes. This is ready to merge.

**Pull request merged**